### DISCUSSION TITLE
New issue format

### DISCUSSION THREAD

**Discussion by kgodey** [mathesar-foundation/mathesar#763] [Maintainer discussions]
So far, we've been creating separate GitHub issues for design, backend, and frontend work for new features. 

I'd like to switch to a new format where we have a single issue for a given feature. It would first get assigned to @ghislaineguerin for design work, once that's complete, it would be assigned to whoever's working on the backend, and after that's complete, it would go to the frontend engineer who would then close it.

I think this would have several benefits:
- Centralizes the spec for each feature in a single place so we can all be better synced
- Encourages us to cut features into smaller pieces, like @seancolsen's [suggestion to split up Foreign Key constraint work](https://github.com/centerofci/mathesar/discussions/744#discussioncomment-1545262)
- Less work in terms of issue creation

What do you all think?

**Comment by pavish** [comment]
@kgodey I think this is a great idea

**Comment by seancolsen** [comment]
Sounds good :+1:

**Comment by mathemancer** [comment]
I really like the idea of integrating a whole vertical slice for features.  I think you're right that it'll encourage us to make sure things aren't sprawling too wide.

**Comment by ghislaineguerin** [comment]
We have already discussed this, and I think it's a great idea as well.
